src/waveform.js

```diff
@@ -20,21 +20,32 @@ export default class Waveform {
     this.boundMonitorAudioTimeupdate = Waveform.monitorAudioTimeupdateCallback.bind(
       this
     );
+
+    this.boundApplyConfig = Waveform.applyConfig.bind(this);
+
+    try {
+      this.port = chrome.runtime.connect(null, { name: "bandcamplabelview" });
+    } catch (e) {
+      if (e.message.includes("Error in invocation of runtime.connect")) {
+        this.log.error(e);
```

Comment:
Looks like this try/catch is to handle a message that is ok to skip... but why? Is it worth including a log line for the reason?

Reply:
Moving to https://github.com/sabjorn/BandcampEnhancementSuite/issues/85